<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalanceBeamWidget</class>
 <widget class="QWidget" name="BalanceBeamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font-family: &quot;Segoe UI&quot;;
	font-size: 16px;
	color: #033f63;
}

QPushButton {
	background-color: #033f63;
	color: #d9d9d9;
	border-radius: 4px;
	padding: 8px 16px;
}

#sampling_rate_label, #input_label, #filter_label, #mid_corner_label, #low_corner_label, #high_corner_label {
	font-size: 20px;
	text-align: center;
	color: #98012e;
}

background-color: #d9d9d9;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="home_btn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="basic_operation.qrc">
       <normaloff>:/assets/assets/home_icon.png</normaloff>:/assets/assets/home_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="graph" native="true">
     <widget class="QLabel" name="graph_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>682</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Ball Position Vs. Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="kp_input">
       <property name="text">
        <string>0.8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kp_output">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="ki_input">
       <property name="text">
        <string>0.4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ki_output">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="kd_input">
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kd_output">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="n_input">
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="n_output">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="send_constants_btn">
     <property name="text">
      <string>Send Constants</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="set_cm_input">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="set_cm_output">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="send_set_point_btn">
     <property name="text">
      <string>Send Set Point</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_step_btn">
     <property name="text">
      <string>Save Step</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_plot_data_btn">
     <property name="text">
      <string>Save Plot Data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="offset_inc_btn">
       <property name="text">
        <string>Offset +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="offset_dec_btn">
       <property name="text">
        <string>Offset -</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pause_btn">
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="basic_operation.qrc"/>
 </resources>
 <connections/>
</ui>
